src/autogluon/cloud/cluster/ray_cluster_manager.py

```diff
@@ -89,7 +90,7 @@ def setup_dashboard(self, port: int = 8265, block: bool = False, **kwargs) -> No
         """
         cmd = f"ray dashboard -p {port} {self.config}"
         if not block:
-            cmd = "nohup " + cmd + "&>/dev/null &"
+            cmd = "nohup " + cmd + " >/dev/null 2>&1 &"
```

Comment:
not safe use of CLI: allows command injection: i.e. pass port to something like `123 | rm -rf some_dir`

Reply:
This command will be called by the user themselves and is not exposed inside the cluster. (There's no autogluon.cloud being installed inside the cluster). Therefore, I think this should be fine